src/Abp.ZeroCore/Authorization/Users/UserTokenExpirationWorker.cs

```diff
@@ -15,18 +15,17 @@ public class UserTokenExpirationWorker<TTenant, TUser> : PeriodicBackgroundWorke
         where TTenant : AbpTenant<TUser>
         where TUser : AbpUserBase
     {
-        private const int IntervalInMilliseconds = 1 * 60 * 60 * 1000; // 1 hour
-
         private readonly IRepository<UserToken, long> _userTokenRepository;
         private readonly IRepository<TTenant> _tenantRepository;
         private readonly IBackgroundJobConfiguration _backgroundJobConfiguration;
         private readonly IUnitOfWorkManager _unitOfWorkManager;
+        private TimeSpan _frequency => _backgroundJobConfiguration.UserTokenExpirationFrequency ?? TimeSpan.FromHours(1);
```

Comment:
- We don't need a private getter for this; it's only used once.
- Frequency is the inverse of period. This value is the period.

Reply:
👍